ExpandedStash/mod.js

```diff
@@ -3,6 +3,15 @@ if (D2RMM.getVersion == null || D2RMM.getVersion() < 1.6) {
   return;
 }
 
+const tabNamePersonal = (!config.IsCustomTabNamesEnabled || config.tabNamePersonal === "") ? "@personal" : config.tabNamePersonal;
+const tabNameShared1  = (!config.IsCustomTabNamesEnabled || config.tabNameShared1  === "") ? "@shared"   : config.tabNameShared1;
+const tabNameShared2  = (!config.IsCustomTabNamesEnabled || config.tabNameShared2  === "") ? "@shared"   : config.tabNameShared2;
+const tabNameShared3  = (!config.IsCustomTabNamesEnabled || config.tabNameShared3  === "") ? "@shared"   : config.tabNameShared3;
+const tabNameShared4  = (!config.IsCustomTabNamesEnabled || config.tabNameShared4  === "") ? "@shared"   : config.tabNameShared4;
+const tabNameShared5  = (!config.IsCustomTabNamesEnabled || config.tabNameShared5  === "") ? "@shared"   : config.tabNameShared5;
+const tabNameShared6  = (!config.IsCustomTabNamesEnabled || config.tabNameShared6  === "") ? "@shared"   : config.tabNameShared6;
+const tabNameShared7  = (!config.IsCustomTabNamesEnabled || config.tabNameShared7  === "") ? "@shared"   : config.tabNameShared7;
+
```

Comment:
Could you run `prettier` on this please? Should work automatically if you edit in VS Code due to the `package.json` rules. It keeps formatting consistent across the repro.